src/components/SkillInfo.tsx

```diff
@@ -85,13 +86,14 @@ const SkillInfo: React.VFC<
   };
 
   const [skillLevel, setSkillLevel] = useState(maxLevel);
+  const extendRange = levels[skillLevel - 1].blackboard.find(kv => kv.key === "ability_range_forward_extend")?.value;
```

Comment:
Oh, that's cursed–the *descriptions* can have all-caps `ability_range_forward_extend`, but the _blackboard_ never seems to have an all-caps version of the key. Well, I guess this is fine–do we want to do `kv.key.toLowerCase() === "ability_range_forward_extend"` "just in case" or is that overkill?